### ISSUE TITLE
Doesn't seem to work at all

### ISSUE THREAD

**Issue by developdeez** [itgalaxy/webpack-modernizr-loader#103]
Followed the instructions and no result in the html. must be outdated

**Comment by cascornelissen** [comment]
I've been searching/debugging for a few hours today so I'm hoping this helps you or someone else. All Webpack loaders and plugins that handle Modernizr builds don't seem to be updated for Webpack 5 or are no longer maintained at all anymore.

There's [a guide](https://webpack.js.org/loaders/val-loader/#modernizr) within the official Webpack documentation on how to set this up with [`val-loader`](npmjs.com/package/val-loader) but it requires quite a big custom implementation. Since I needed this in multiple projects I have abstracted this away into (shameless plug) [`val-loader-modernizr`](http://npmjs.com/package/val-loader-modernizr), feel free to use it if you're using Webpack 5 ✌🏼

**Comment by alexander-akait** [comment]
Yep, original solution here https://github.com/webpack-contrib/val-loader#modernizr
